setup/bootstrap.sh

```diff
@@ -2,18 +2,36 @@
 #########################################################
 # This script is intended to be run like this:
 #
-#   curl https://mailinabox.email/setup.sh | sudo bash
+#   wget https://mailinabox.email/setup.sh -qO - | sudo bash -s
```

Comment:
I suspect new users will be confused by having two ways to do it. Perhaps adjust the title section or add an "OR" here, to make it clear both these commands achieve the same thing.

Reply:
This is in a comment that I doubt new users will read, but I will add an "or" if you think it will be clearer.